src/pineko/check.py

```diff
@@ -84,7 +84,7 @@ def is_fonll_b(fns, lumi):
     """
     for lists in lumi:
         for el in lists:
-            if not (10 < abs(el[1]) < 17):
+            if (not (10 < abs(el[0]) < 17)) and (not (10 < abs(el[1]) < 17)):
```

Comment:
Since I'm writing on this PR in any case: instead of repeating twice the condition, make a brief `is_lepton` function (even in this function scope, or top level in the file, as you wish).